projects/canopy/src/lib/brand-icon/brand-icons.stories.ts

```diff
@@ -120,21 +103,81 @@ const colours = [
   '--color-poppy-red-dark',
 ];
 
-export const standard = () => ({
-  template: `
-    <lg-swatch-brand-icon [size]="size" [colour]="colour" [specificColour]="specificColour"></lg-swatch-brand-icon>
-  `,
-  props: {
-    size: select('size', sizes, 'xs'),
-    colour: select(
-      'example of applying colour globally',
-      colours,
-      '--color-dandelion-yellow',
-    ),
-    specificColour: select(
-      'example of applying colour specifically to an icon',
-      colours,
-      '--color-super-blue',
-    ),
+export default {
+  title: 'Components/Brand Icon',
+  decorators: [
+    moduleMetadata({
+      declarations: [SwatchBrandIconComponent, LgBrandIconComponent],
+    }),
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: notes,
+      },
+    },
+  },
+  argTypes: {
+    size: {
+      options: sizes,
+      description: 'The size of the icon',
+      table: {
+        type: {
+          summary: sizes,
+        },
+        defaultValue: {
+          summary: 'sm',
+        },
+      },
+      control: {
+        type: 'select',
+      },
+    },
+    colour: {
+      options: colours,
+      description:
+        'The primary colour of icons globally. Can be changed by overriding the `--brand-icon-fill-primary` CSS variable.',
+      table: {
+        defaultValue: {
+          summary: '--color-dandelion-yellow',
+        },
+      },
+      control: {
+        type: 'select',
+      },
+    },
+    specificColour: {
+      options: colours,
+      description: 'The colour of a specific icon, using the `colour` input',
+      table: {
+        type: {
+          summary: colours,
+        },
+        defaultValue: {
+          summary: '--color-dandelion-yellow',
+        },
+      },
+      control: {
+        type: 'select',
+      },
+    },
   },
+};
+
+const template = `
+  <lg-swatch-brand-icon [size]="size" [colour]="colour" [specificColour]="specificColour"></lg-swatch-brand-icon>
+`;
+const brandIconsTemplate: Story<LgBrandIconComponent> = (args: LgBrandIconComponent) => ({
+  props: args,
+  template: template,
```

Comment:
I would personally move the template straight into here since it's only one line, no?

Reply:
> @owensgit something doesn't look right in the template code 🤔 it gets displayed like: ![Screenshot 2021-11-03 at 11 42 38](https://user-images.githubusercontent.com/8397116/140054147-daf6c275-3b98-442d-9b5d-4e8a568c449a.png)

Ah yes, I'll take a look into that, thanks :)